src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/MultiMathExpressionConverter.shared.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Xamarin.CommunityToolkit.Core;
+using Xamarin.CommunityToolkit.Extensions.Internals;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class MultiMathExpressionConverter : MultiValueConverterExtension, IMultiValueConverter
```

Comment:
what is the purpose of this converter? Is it the same as a MathExpressionConverter but supposed to have several parameters? 

if yes, I believe, we can leave only MathExpressionConverter (where we can check if value an array or a single number)

Reply:
yes, it supposed to have several params and to use MultiBinding. And I think MultiBinding is more comfortable to use in XAML. For example if developer create the function who depends on Height and Width of element and you can't set the bindings without MultiBinding, I guess.